missing change of bio->bi_disk->private_data;

opensuse don't have blk_queue_update_readahead
	upstream commit c2e4cd57cfa1f
opensuse don't have revalidate_disk_size
	upstream commit 659e56ba864d3

diff -Naur drbd-9.0.26~0+git.9114a038.orig/drbd/drbd_req.c drbd-9.0.26~0+git.9114a038/drbd/drbd_req.c
--- drbd-9.0.26~0+git.9114a038.orig/drbd/drbd_req.c	2020-12-08 22:03:48.629954854 +0800
+++ drbd-9.0.26~0+git.9114a038/drbd/drbd_req.c	2020-12-09 10:45:04.390742754 +0800
@@ -2196,7 +2196,12 @@
 
 blk_qc_t drbd_submit_bio(struct bio *bio)
 {
+#ifdef COMPAT_HAVE_SUBMIT_BIO
+	struct drbd_device *device = bio->bi_disk->private_data;
+#else
 	struct drbd_device *device = (struct drbd_device *) q->queuedata;
+#endif
+
 #ifdef CONFIG_DRBD_TIMING_STATS
 	ktime_t start_kt;
 #endif
